crates/router/src/connector/stripe.rs

```diff
@@ -1929,3 +1952,484 @@ impl services::ConnectorRedirectResponse for Stripe {
             ))
     }
 }
+
+impl api::Payouts for Stripe {}
+#[cfg(feature = "payouts")]
+impl api::PayoutCancel for Stripe {}
```

Comment:
why `api::Payouts for Stripe {}` is not under feature flag?

Reply:
We have `pub trait Connector:` which cannot implement feature based traits; so Payouts is implemented for all the connectors